packages/fluent/scss/treelist/_layout.scss

```diff
@@ -44,6 +44,11 @@
         tr.k-footer + tr.k-footer td {
             border-top-width: 0;
         }
+
+        .k-i-collapse,
+        .k-i-expand {
```

Comment:
same as first comment

Reply:
The icons are with 16/16 size and I don't see a specified line height in the design.

Reply:
I was referring to my comment regarding using specific icon classes here